contracts/exchange/IbetSecurityTokenDVP.sol

```diff
@@ -588,6 +588,36 @@ contract IbetSecurityTokenDVP is Ownable, IbetExchangeInterface {
         return success;
     }
 
+    /// @notice 部分的に残高を引き出しする
+    /// @dev 決済で拘束されているものは引き出しされない
+    /// @param _token トークンアドレス
+    /// @param _value 引き出し数量
+    /// @return 処理結果
+    function withdrawPartial(
+        address _token,
+        uint _value
+    ) public returns (bool) {
+        uint256 balance = balanceOf(msg.sender, _token);
+
+        require(
+            balance >= _value,
+            ErrorCode.ERR_IbetSecurityTokenDVP_withdraw_260501
+        );
```

Comment:
Can we have new error code of `260601` as `withdrawPartial` one?

https://github.com/BoostryJP/ibet-SmartContract/blob/dev-24.9/docs/Errors.md#withdraw-2605xx